wikidatarefisland/pipes/__init__.py

```diff
@@ -1,4 +1,14 @@
+from .pipe_one import ItemExtractorPipe
 from .abstract_pipe import AbstractPipe
+from .statement_filters import referenced_statement_excluder, external_id_statement_excluder, \
```

Comment:
I wonder whether these should go in their own submodule for cohesion's sake?

Reply:
Yeah, I think it should go to data_model part.